newrelic/api/transaction.py

```diff
@@ -1438,6 +1438,18 @@ def accept_distributed_trace_headers(self, headers, transport_type="HTTP"):
                             data.update(tracestate_data)
                         else:
                             self._record_supportability("Supportability/TraceContext/TraceState/InvalidNrEntry")
+                    elif not payload and (tracestate == self.tracestate):
+                        self._record_supportability("Supportability/TraceContext/TraceState/NoNrEntry")
+                        self._record_supportability("Supportability/TraceContext/TraceState/OtelEntry")
+                        try:
+                            vendors["sa"] = True if vendors.get("sa").lower() == "true" else False
```

Comment:
We might not even need this since the sampled flag already gets picked up by the traceparent.


```suggestion
                            vendors.pop("sa", None)
```